x-pack/plugins/observability_solution/inventory/public/components/inventory_page_template/index.tsx

```diff
@@ -32,30 +41,47 @@ export function InventoryPageTemplate({ children }: { children: React.ReactNode
     toggleWelcomedModal();
   };
 
+  const { value = { hasData: false }, loading } = useInventoryAbortableAsync(
```

Comment:
```suggestion
  const { value = { hasData: false }, loading: hasDataLoading } = useInventoryAbortableAsync(
```

Reply:
updated it here https://github.com/elastic/kibana/pull/193755/commits/21bf56497bbc8b929c6371558cb7339b608e8fca